### DISCUSSION TITLE
Jellyseerr Compatibility on Linux

### DISCUSSION THREAD

**Discussion by willzoy** [itsToggle/plex_debrid#545] [Q&A]
I've been attempting to set up a Jellyfin server using plex_debrid pulling content requests using Jellyseerr on a Raspberry Pi 4 for the past couple of days. I've manager to overcome many stumbling blocks but I've reached a point of complete frustration. 

When choosing a Content Service, I am prompted to choose between Plex, Trakt, and Overseerr. Despite the documentation mentioning Jellyseerr, it isn't an option. As Jellyseerr is a fork of Overseerr, I attempted to set it up through that option however it won't accept any API key I provide it from Jellyseerr.

Is there a step I'm missing here, or would I be better off switching to a Plex server and running it off of Plex's content & library services?

**Comment by epthewizard** [comment]
> I've been attempting to set up a Jellyfin server using plex_debrid pulling content requests using Jellyseerr on a Raspberry Pi 4 for the past couple of days. I've manager to overcome many stumbling blocks but I've reached a point of complete frustration. 
> 
> 
> 
> When choosing a Content Service, I am prompted to choose between Plex, Trakt, and Overseerr. Despite the documentation mentioning Jellyseerr, it isn't an option. As Jellyseerr is a fork of Overseerr, I attempted to set it up through that option however it won't accept any API key I provide it from Jellyseerr.
> 
> 
> 
> Is there a step I'm missing here, or would I be better off switching to a Plex server and running it off of Plex's content & library services?

I had it setup once and it worked fine. But I did it straight through the config file.
